docs/INSTALL-COMPLETE-GUIDE.md

```diff
@@ -0,0 +1,145 @@
+# How to set up a Phoniebox from scratch
+
+## What you need
+
+All parts marked with a star (*) are optional but improve the overall experience. All linked components are examples but have proven to work together. You are free to choose different equipment.
+
+1. [Micro SD Card](https://www.amazon.de/gp/product/B073JWXGNT) (e.g. 32 GB)
+1. [Raspberry Pi](https://www.amazon.de/gp/product/B07BDR5PDW) (Model 1, 2, 3, 4, Zero)
+1. [USB RFID Reader](https://www.amazon.de/gp/product/B018OYOR3E)
+1. [RFID Chips](https://www.amazon.de/gp/product/B078WRNLQZ) or [RFID Cards](https://www.amazon.de/gp/product/B00REFN24A)
+1. [Speakers with 3.5mm jack](https://www.amazon.de/gp/product/B00JRW0M32)
+1. [USB Sound Card](https://www.amazon.de/gp/product/B00C7LXUDY) * - [Alternative](https://www.amazon.de/gp/product/B00IRVQ0F8)
+1. [Ground Loop Isolator](https://www.amazon.de/gp/product/B01LX0H29W) *
+
+You'll need a few other things for a one time set up only.
+
+1. Second computer (Linux, Mac or Windows)
+1. USB Mouse and USB Keyboard
+1. Micro SD Card Reader
+1. Screen with HDMI connection
+
+Alternative: If you are able to connect your Raspberry Pi via a wired network interface (LAN), you can set it up via terminal (SSH) only.
```

Comment:
It's also possible to set up a Pi with WiFi and ssh, e.g. https://raspberrypi.stackexchange.com/questions/10251/prepare-sd-card-for-wifi-on-headless-pi/57023#57023

I think that's worth mentioning, too.

Reply:
Added